debian/rules

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+
+%:
+	dh $@ --with python2 --builddirectory=build
+
+override_dh_auto_configure:
+	export QT_SELECT=5
+	dh_auto_configure -- -DUSE_QT5=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_SHIBBOLETH_SUPPORT=ON
```

Comment:
`USE_QT5` has been removed for a while (we dropped qt4 and only supports qt5 since 5.1.0).